### ISSUE TITLE
Create revised error handling on Property Insert scripts in python

### ISSUE THREAD

**Issue by edarsow** [TechnologyRediscovery/codenforce#29]
Per Zooming with Mr. Snapper, let's create error handling that results in a parcel ID to get written to an error file which could then, in a later issue, get slurped back up in python and dealt with. 

For example: Contents of error file could look like this
00034H323200000,ERR30.OWNER_NAME_BUT_NO_ADDRESS
00023R322230000,ERR31.NO_OWNER_NAME_IN_COUNTY
...
Create a first pass sensible way to document the error codes by number, such as a table in the database!! Watch out for a new issue on this!

**Comment by edarsow** [comment]
Mr. Snapper and I spoke via phone: looks good!

**Comment by snapperVibes** [comment]
I overhauled error handling for insertDataFromCntyUsingParid today.
Custom errors are raised from the new _exceptions.py file.
Exceptions are as follows:

**MalformedDataError**
The base class for errors resulting from Malformed Data.

**MalformedAddressError**
Thrown when scraped address HTML doesn't fit expected format and we can't further identify the error.

**MalformedStateError**, **MalformedZipcodeError**, and **MalformedLotAndBlockError**
Self explanatory errors subclassing the MalformedAddressError

**MalformedOwnerError**
This is not subclassed into FirstNameErrors, LastNameErrors, etc as parsing names is challenging to begin with. We need to discuss how the compound last name flag works in the database so the script handles compound names / errors the same way as other parts of the system.

All uses of 
```
try:
    example_func()
except:
    Exception
```
have been made to accept one of the above errors

The standard library's `logging` module is used to create the .log file.


Todo:
---
I want to be clever and have a logging function called automatically when an exception is raised by putting the logging in the exceptions `__init__` method, but I haven't found an intuitive implementation yet.

Closing Notes
---
I believe I will have much more time to devote to this project in the coming weeks.
I am not sure when to merge my work with the recovered branch. As of now, I do not plan to merge until I have everything working how I want it to.
